irohad/model/execution/impl/command_executor.cpp

```diff
@@ -814,33 +794,33 @@ namespace iroha {
       auto src_account_asset = queries.getAccountAsset(
           transfer_asset.src_account_id, transfer_asset.asset_id);
       if (not src_account_asset.has_value()) {
-        return makeExecutionError((boost::format("asset %s is absent of %s")
-                                   % transfer_asset.asset_id
-                                   % transfer_asset.src_account_id)
-                                      .str());
+        return makeExecutionResult((boost::format("asset %s is absent of %s")
+                                    % transfer_asset.asset_id
+                                    % transfer_asset.src_account_id)
+                                       .str());
       }
 
       AccountAsset dest_AccountAsset;
       auto dest_account_asset = queries.getAccountAsset(
           transfer_asset.dest_account_id, transfer_asset.asset_id);
       auto asset = queries.getAsset(transfer_asset.asset_id);
       if (not asset.has_value()) {
-        return makeExecutionError((boost::format("asset %s is absent of %s")
-                                   % transfer_asset.asset_id
-                                   % transfer_asset.dest_account_id)
-                                      .str());
+        return makeExecutionResult((boost::format("asset %s is absent of %s")
+                                    % transfer_asset.asset_id
+                                    % transfer_asset.dest_account_id)
+                                       .str());
       }
       // Precision for both wallets
       auto precision = asset.value().precision;
       if (transfer_asset.amount.getPrecision() != precision) {
-        return makeExecutionError(
+        return makeExecutionResult(
             (boost::format("precision %d is wrong") % precision).str());
       }
       // Get src balance
       auto src_balance = src_account_asset.value().balance;
       auto new_src_balance = src_balance - transfer_asset.amount;
       if (not new_src_balance.has_value()) {
-        return makeExecutionError("not enough assets on source account");
+        return makeExecutionResult("not enough assets on source account"s);
```

Comment:
Better to write account id in the result explicitly.

Reply:
Agreed, will add